app/apps/cases/views.py

```diff
@@ -185,22 +185,11 @@ def list(self, request):
             queryParams = dict(
                 (param_translate.get(k, k), v) for k, v in queryParams.items()
             )
-            queryParams.update(
-                {
-                    "open_cases": True,
-                    "page_size": 1000,
-                    "task": ["task_uitvoeren_leegstandsgesprek", "task_create_visit"],
-                }
-            )
-            response = requests.get(
-                url,
-                params=queryParams,
-                timeout=60,
-                headers=get_headers(get_keycloak_auth_header_from_request(request)),
-            )
-            response.raise_for_status()
-
-            result = response.json().get("results", [])
+            result = []
+            for task in settings.AZA_ALLOWED_TASK_NAMES:
```

Comment:
Can you add a comment why you are doing this for each individual task rather than all at once?